src/vcr_clj/cassettes.clj

```diff
@@ -28,8 +28,15 @@
     (binding [*out* writer]
       (prn cassette))))
 
+(defn- reattach-http-meta
+  [cassette]
+  (let [set-http-meta #(vary-meta % assoc :type :vcr-clj.clj-http/serializable-http-request)
+        update-episode #(update % :return set-http-meta)]
+    (update cassette :calls #(mapv update-episode %))))
+
 ;; TODO: use clojure.edn?
 (defn read-cassette
   [name]
-  (with-open [r (java.io.PushbackReader. (io/reader (cassette-file name)))]
-    (edn/read {:readers data-readers} r)))
+  (let [cassette (with-open [r (java.io.PushbackReader. (io/reader (cassette-file name)))]
+                   (edn/read {:readers data-readers} r))]
+    (reattach-http-meta cassette)))
```

Comment:
This problem would be better solved in the `vcr-clj.clj-http` namespace, if possible. The `print-method` for `::serializable-http-request` in that namespace could be enhanced so that http requests are recognized when reading, maybe with some help from the data-readers in the serialization namespace.

The reason for this is that everything in `vcr-clj` outside of the `clj-http` namespace is meant to be generic, and so attaching this metadata at this point in the code is inappropriate since it would be applied to _all_ function calls, not just http requests. (there is currently a test failure because of this problem)